docs/install.md

```diff
@@ -29,26 +28,27 @@ For more details about MongoDB extension and library for PHP check [here][PHPMon
 [php]: http://php.net/downloads.php
 [mongo]: http://php.net/mongo
 [mongodb]: http://php.net/mongodb
-[php72]: https://www.drupal.org/node/2936045
 [drush9]: https://www.drupal.org/project/mongodb/issues/2986785
 [PHPMongoDBext]: http://php.net/mongodb
 [MongoDBMac]: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/
 [MongoDBLinux]: https://docs.mongodb.com/manual/administration/install-on-linux/
 [MongoDBWindows]: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/
+[removedhttp]: https://docs.mongodb.com/manual/release-notes/3.6-compatibility/#http-interface-and-rest-api
 
-If MongoDB 3.0 to 4.0 is installed on `localhost:27017` and `mongod` was started
+If MongoDB 3.0 to 3.5[^2] is installed on `localhost:27017` and `mongod` was started
 with the `–httpinterface` option, you may view the web admin interface:
 
     http://localhost:28017/
 
-##Settings Configuration
-* Download the module package, as per
-  [Installing contributed modules (Drupal 8)][install]
+[^2]: This option is [deprecated from 3.6 version][removedhttp].
+
+## Settings Configuration
```

Comment:
skip line between title and text

Reply:
Same as above.... could you please explain what is expected here?

Reply:
Yes, in most of the pages, any time there is a new heading (#, ## or ###) and the previous line is not itself a higher level heading (# for ##, ## for ###), the first heading is preceded by two blank lines, not just one. This doesn't change the output, but makes the source Markdown easier to read.